subliminal/cli.py

```diff
@@ -722,3 +722,7 @@ def download(
 
     if verbose == 0:
         click.echo(f"Downloaded {plural(total_subtitles, 'subtitle')}")
+
+
+if __name__ == '__main__':
+    subliminal()
```

Comment:
this should be in a separate commit, if anything

Reply:
also, if this was just a temporary launcher thingy for development, you can achieve the same with `python3 -m subliminal`

Reply:
I tried that, and I get " No module named subliminal.__main__; 'subliminal' is a package and cannot be directly executed" (I'm running from the virtual env, after installing dependencies)

Anyway, no problem, before I finish this PR I will reverse the change

Reply:
You can install subliminal in editable mode with `pip install '.[dev]' -e`, even in a virtual env. Like that you can install subliminal and run it's CLI from the terminal, and at the same time modify the files and see the modification in the installed package.